packages/devtools_extensions/lib/src/api/model.dart

```diff
@@ -53,3 +53,79 @@ class DevToolsExtensionEvent {
 
 /// A void callback that handles a [DevToolsExtensionEvent].
 typedef ExtensionEventHandler = void Function(DevToolsExtensionEvent event);
+
+class ShowNotificationExtensionEvent extends DevToolsExtensionEvent {
```

Comment:
Are these classes accessible to developers? If so, we need documentation for them.

Reply:
moved to a file that is not exported to the user. Users should send events through the ExtensionManager. We can always expose later if we need to.

Reply:
Update: I missed that we use these classes from devtools_app to parse, so made them public and documented in model.dart